__tests__/integration/app.test.js

```diff
@@ -231,9 +231,68 @@ describe("GET /api/articles/:article_id/comments", () => {
 			.expect(200)
 			.then(({ body }) => {
 				const { comments } = body;
-        expect(comments).toBeInstanceOf(Array);
+				expect(comments).toBeInstanceOf(Array);
 				expect(comments).toHaveLength(0);
-        expect(comments).toEqual([])
+				expect(comments).toEqual([]);
+			});
+	});
+});
+
+describe("POST /api/articles/:article_id/comments", () => {
+	test("201: responds with the newly posted comment object ", () => {
+		const newComment = {
+			username: "butter_bridge",
+			body: "this is my comment for the article",
+		};
+		return request(app)
+			.post("/api/articles/1/comments")
+			.send(newComment)
+			.expect(201)
+			.then(({ body }) => {
+				expect(body.comment).toMatchObject({
+					comment_id: expect.any(Number),
+					article_id: 1,
+					author: "butter_bridge",
+					votes: 0,
+					created_at: expect.any(String),
+				});
```

Comment:
This expect statement is missing the body key which should be 

`body: "this is my comment for the article",`

From the given object